index.html

```diff
@@ -1,31 +1,71 @@
 <html>
-  <script src="try-slang.js"> </script>
+<style>
+textarea {
+  width: 100%;
+  height: 100%;
+  padding: 12px 20px;
+  box-sizing: border-box;
+  border: 2px solid #ccc;
+  border-radius: 4px;
+  background-color: #121212;
+  color: #ffffff;
+  font-size: 16px;
+  resize: none;
+}
+</style>
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+  <script src="https://unpkg.com/monaco-editor@latest/min/vs/loader.js"></script>
+  <script>
+          require.config({ paths: { 'vs': 'https://unpkg.com/monaco-editor@latest/min/vs' }});
+          window.MonacoEnvironment = { getWorkerUrl: () => proxy };
+          let proxy = URL.createObjectURL(new Blob([`
+              self.MonacoEnvironment = {
+                  baseUrl: 'https://unpkg.com/monaco-editor@latest/min/'
+              };
+              importScripts('https://unpkg.com/monaco-editor@latest/min/vs/base/worker/workerMain.js');
+          `], { type: 'text/javascript' }));
+  </script>
+  <script src="util.js"></script>
+  <script src="pass_through.js"></script>
+  <script src="compute.js"></script>
+  <script src="try-slang.js"></script>
   <script src="slang-wasm.js"></script>
   <head>
     <title>Try Slang!</title>
   </head>
   <body>
-    <h1>Input (Slang)</h1>
-    <textarea id="input" rows="20" cols="90">
-RWStructuredBuffer<int> outputBuffer;
+    <div class="w3-container w3-black w3-teal">
 
-void writeArray(inout float3 a[4])
-{
-    a[0] = float3(1, 1, 1);
-    a[1] = float3(1, 1, 1);
-    a[2] = float3(1, 1, 1);
-    a[3] = float3(1, 1, 1);
-}
+      <div class="w3-row-padding">
+        <div id="WebGPU-status-bar" style="width: 100%; height: 50px;"></div>
```

Comment:
we should avoid doing inline styles like this, instead put the style in the css file as:
```
#WebGPU-status-bar {
    width: 100%;
    height: 50px;
}
```

Also consider using the `flex` layout to make the page resizable.